feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenEvent.kt

```diff
@@ -0,0 +1,6 @@
+package com.epmedu.animeal.home.presentation
+
+internal sealed interface HomeScreenEvent {
```

Comment:
Why exactly `interface` here? Why not plain sealed class?

Reply:
@Makentoshe Since I don't need sealed class constructor